src/editables/__init__.py

```diff
@@ -53,18 +29,21 @@ def build_editable(location, expose=None, hide=None):
     location = Path(location)
 
     if expose:
-        for e in expose:
```

Comment:
similar here I'm tempted to remove other validations too

Reply:
I disagree, I'd want to validate the inputs. We don't work properly if the caller says "include foo" and "exclude bar.baz" and I don't think that should go silently.

Reply:
I did not say things should not be validated. But I consider validation the job of the backend that already does the collection not of this library.